core/src/main/java/com/facebook/testing/screenshot/internal/ScreenshotImpl.java

```diff
@@ -59,6 +59,7 @@
   private Canvas mCanvas = null;
   private ViewHierarchy mViewHierarchy;
   private boolean mEnableBitmapReconfigure = (Build.VERSION.SDK_INT >= 19);
+  private boolean mDump = true;
```

Comment:
mShouldDumpMetadata

Also, a one line javadoc would be helpful since this is an edge case and most people wouldn't be thinking of this while reading this code